paddle/fluid/operators/ngraph/ops/top_k_op.h

```diff
@@ -0,0 +1,51 @@
+/*Copyright (c) 2018 PaddlePaddle Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License. */
+
+#ifdef PADDLE_WITH_NGRAPH
```

Comment:
could you try to move this to cmakelist?

Reply:
I think it's no need to add `PADDLE_WITH_NGRAPH` here. Since `ngraph_ops.h` is only include in `ngraph_bridge.cc`, and `ngraph_bridge.cc` is compiled `WITH_NGRAPH`.

Reply:
@mozga-intel Please fix this in next PR.